app/assets/javascripts/initializers/initializeSheCodedArticles.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+// This file is specifically used for SheCoded,
+// We can try to make it more general for campaigns in the future once we know more (classnames, params).
+
+function renderArticles(articles, articleElement ) {
```

Comment:
small typo, the space

Reply:
i feel like eslint should have caught this, right? 👀

Reply:
Don't know, maybe!  We do have 2 sets of "eslint config files", one for these "legacy" JS files and one for the component oriented Preact files. Maybe their configurations are not aligned

cc @nickytonline for when he gets back from the holidays